src/network_analysis.jl

```diff
@@ -420,32 +462,41 @@ end
 ```
 """
 function deficiency(rn::ReactionSystem)
+    # Precomputes required information. `conservationlaws` caches the conservation laws in `rn`.
     nps = get_networkproperties(rn)
     conservationlaws(rn)
     r = nps.rank
     ig = incidencematgraph(rn)
     lc = linkageclasses(rn)
+
+    # Computes deficiency using its formula. Caches and returns it as output.
     nps.deficiency = Graphs.nv(ig) - length(lc) - r
     nps.deficiency
 end
 
-# Used in the subsequent function.
+# For a linkage class (set of reaction complexes that form an isolated sub-graph), and some
+# additional information of the full network, find the reactions, species, and parameters 
+# that constitute the corresponding sub-reaction network.
 function subnetworkmapping(linkageclass, allrxs, complextorxsmap, p)
-    rxinds = sort!(collect(Set(rxidx for rcidx in linkageclass
-    for rxidx in complextorxsmap[rcidx])))
-    rxs = allrxs[rxinds]
-    specset = Set(s for rx in rxs for s in rx.substrates if !isconstant(s))
-    for rx in rxs
+    # Finds the reactions that are part of teh sub-reaction network.
+    rxinds = sort!(collect(Set(
+        rxidx for rcidx in linkageclass for rxidx in complextorxsmap[rcidx])))
```

Comment:
Old:
```julia
rxinds = sort!(collect(Set(rxidx for rcidx in linkageclass
for rxidx in complextorxsmap[rcidx])))
```
and new
```julia
rxinds = sort!(collect(Set(
    rxidx for rcidx in linkageclass for rxidx in complextorxsmap[rcidx])))
```
are identical, but I moved the new line a bit to make the formater less bad (i.e. starting `for` without indent)